docs/content/deployment/guides/aws.mdx

```diff
@@ -170,6 +186,6 @@ resources:
       s3_prefix: good/prefix-for-files-
 ```
 
-The resource uses `boto` under the hood, so if you are accessing your private buckets, you will need to provide the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables or follow one of the other boto authentication methods.
+The resource uses `boto` under the hood. If you're accessing your private buckets, you will need to provide the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables or follow one of the other boto authentication methods.
```

Comment:
we could link to https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#configuring-credentials in the last bit